cmd/kind/kind.go

```diff
@@ -54,6 +56,7 @@ func Main() {
 		TimestampFormat: "0102-15:04:05",
 	})
 	if err := Run(); err != nil {
-		log.Fatal(err)
+		os.Stderr.WriteString(err.Error())
+		os.Exit(-1)
```

Comment:
Why the switch from log.fatal?

Reply:
part of #23, cli tools should print an error to stdout and exit non-zero instead of `FATA ....` ... I'm going to reserve logging for actual debug info soon and use println and friends for actual user oriented output.

failing to run needs to be user oriented rather than logging. if this were eg a server I'd still do fatal